packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.scss

```diff
@@ -272,44 +284,54 @@ $ms-choiceField-transition-timing: cubic-bezier(.4, 0, .23, 1);
             display: inline-block;
           }
         }
-        .ms-ChoiceField-iconWrapper {
-          font-size: $ms-font-size-su;
-          line-height: $ms-font-size-su;
-        }
       }
 
       .ms-ChoiceField-labelWrapper {
+        $lineHeight: 16px;
+
         display: inline-block;
         position: relative;
-        margin: 10px 0 0 0;
-        padding: 0 24px;
+        margin: 4px 0 0 0;
+        height: $lineHeight * 2;
+        line-height: $lineHeight;
```

Comment:
I am removing line-height from the root, and as per Design we need to reduce line-height in this case from default so that we can fit in two lines of text in that 96 by 96 px icon ChoiceField box.